### PR TITLE
Feature/card for view controller

### PR DESCRIPTION
**Pull request by attila-at-hyper**

### CODE REVIEW

**Comment by kostiakoval** [comment]
Have you thought make a container view controller, that will work with viewConteollers instead of views?

**Comment by attila-at-hyper** [comment]
@kostiakoval: yes, it's on my to do list (but haven't considered it to be a high priority issue)

**Comment by kostiakoval** [comment]
:+1:

File: Source/CardStackController.m [hunk 1]
```diff
@@ -325,6 +325,31 @@ - (void)cardTitlePanDidFinish:(CardView *)card withVelocity:(CGPoint)velocity {
...
+
+    return card;
+}
+
+- (NSUInteger)indexForViewController:(UIViewController *)viewController {
+    NSUInteger index;
+
+    for (NSUInteger index = 0; index < self.viewControllers.count; index++) {
+        if ([[self.viewControllers objectAtIndex:index] isEqual:viewController]) {
+            break;
```
Comment (3lvis) [inline Source/CardStackController.m]: This method doesn't seem to be doing anything

Comment (attila-at-hyper) [inline Source/CardStackController.m]: @elvis: if you mean `indexForViewController:', then please have a look at the commit below.

**Comment by kostiakoval** [comment]
I also wanted to suggest such solution. You can even to smt. like that

```
__block NSInteger foundIndex;

[self.viewControllers enumerateUsingBlock...
   foundIndex = index;
   *stop = [currentViewController isEqual:viewController]
];
```

**Comment by 3lvis** [comment]
@kostiakoval nice!

**Comment by attila-at-hyper** [comment]
To be honest I don't see why my solution is quite strange, but I'm happy to change it. :) See next commit.

**Comment by kostiakoval** [comment]
You are hardcore C guy :)

**Comment by attila-at-hyper** [comment]
@kostiakoval: yes, I am (though I'm not sure if it's a good or bad :))

**Comment by kostiakoval** [comment]
It's good, definitely!! @attila-at-hyper

**Pull request merged**